### PR TITLE
Bump scanpy and scvelo version

### PR DESCRIPTION
**Pull request by michalk8**
Implements #141 
Will have to deal with writing the transition matrix.

### CODE REVIEW

**Comment by michalk8** [comment]
@Marius1311 I've changed the saving location of the transition matrix to be `adata.obs['T_{fwd,bwd}']`, the params are in `adata.uns['T_{fwd,bwd}_params']` - previously, they were in `adata.uns['T_{fwd,bwd}']['params']`.
If everything is fine, I will get this merged and finish #225

**Comment by Marius1311** [comment]
Hi Mike, sorry, just getting to this now!

File: cellrank/tools/kernels/_kernel.py [hunk 1]
```diff
@@ -155,8 +155,8 @@ def write_to_adata(self, key_added: Optional[str] = None) -> None:
         -------
         None
             Updates the underlying :paramref:`.adata` object with the following:
-                - `.uns[:paramref:`T_{fwd, bwd}` _`:paramref:`key_added`]['T']` - transition matrix
-                - `.uns[:paramref:`T_{fwd, bwd}` _`:paramref:`key_added`]['params']` - parameters used for calculation
+                - `.obsp[`'T_{fwd, bwd}'` _`:paramref:`key_added`]` - transition matrix
+                - `.uns[`'T_{fwd, bwd}'` _`:paramref:`key_added` `'_params'`] - parameters used for calculation
```
Comment (Marius1311) [inline cellrank/tools/kernels/_kernel.py:159]: shouldn't that also be `.obsp`?

Comment (Marius1311) [reply cellrank/tools/kernels/_kernel.py:159]: sorry, my bad, didn't read what you said in the PR. Storing the params in `.uns` is fine.

**Comment by michalk8** [comment]
@Marius1311 No worries, I've been keeping myself busy with `_repr_html` for `Lineage`:
2020-07-02T21:28:12+02:00
Do you have any suggestions?

**Comment by michalk8** [comment]
@Marius1311 thanks a lot, merging!

**Comment by Marius1311** [comment]
oh, that representation for the lineage class looks very nice indeed! Will the colors be the associated lineage class colors? That would be nice!

**Pull request merged**